docs/running/cwl.rst

```diff
@@ -10,12 +10,70 @@ Toil has full support for the CWL v1.0.1 specification.
 Running CWL Locally
 -------------------
 
+The `toil-cwl-runner` command provides cwl-parsing functionality using cwltool, and leverages the job-scheduling and
+batch system support of Toil.
+
 To run in local batch mode, provide the CWL file and the input object file::
 
     $ toil-cwl-runner example.cwl example-job.yml
 
 For a simple example of CWL with Toil see :ref:`cwlquickstart`.
 
+Detailed Usage Instructions
+---------------------------
+
+Help information can be found by using this toil command:
+::
+
+    $ toil-cwl-runner -h
+
+A more detailed example shows how we can specify both Toil and cwltool arguments for our workflow:
+::
+
+    $ toil-cwl-runner \
+        --singularity \
+        --jobStore my_jobStore \
+        --batchSystem lsf \
+        --workDir `pwd` \
+        --outdir `pwd` \
+        --logFile cwltoil.log \
+        --writeLogs `pwd` \
+        --logLevel DEBUG \
+        --retryCount 2 \
+        --disableCaching \
+        --maxLogFileSize 20000000000 \
+        --stats \
+        standard_bam_processing.cwl \
+        inputs.yaml
+
+In this example, we set the following options, which are all passed to Toil:
+::
+
+    --singularity: Specifies that all jobs should be run with Singularity containers instead of
+    Docker.
+
+    --jobStore: Path to a folder that already exists, which will contain the Toil jobstore and all
+        related job-tracking information.
+
+    --batchSystem: Use the specified HPC or Cloud-based cluster platform.
+
+    --workDir: The directory where all temporary files will be created for the workflow. This can be referenced as
```

Comment:
```suggestion
    --workDir: The directory where all temporary files will be created for the workflow. A subdirectory of this will be set
    as the $TMPDIR environment variable and is also available using the CWL expression 
```